lib/modules/dosomething/dosomething_organ_donation/dosomething_organ_donation.module

```diff
@@ -22,9 +22,17 @@ function dosomething_organ_donation_menu() {
     'type' => MENU_CALLBACK,
   ];
 
+  $items['organ-donation/post-registration'] = [
+    'page callback' => 'dosomething_organ_donation_post_registration',
+    'access callback' => 'user_access',
+    'access arguments' => array('administer actions'),
+    'type' => MENU_CALLBACK,
+  ];
+
   $items['organ-donation/send-postal-code'] = [
     'page callback' => 'dosomething_organ_donation_send_postal_code',
-    'access callback' => 'user_is_logged_in',
+    'access callback' => 'user_access',
+    'access arguments' => array('administer actions'),
```

Comment:
ah, apologies for not specifying before, but normally we use the `administer modules` permission